go.mod

```diff
@@ -1,3 +1,11 @@
 module github.com/TheAlgorithms/Go
 
 go 1.19
+
+require (
+	github.com/abaron10/Gothon v0.0.2 // indirect
+	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/pmezard/go-difflib v1.0.0 // indirect
+	github.com/stretchr/testify v1.8.1 // indirect
+	gopkg.in/yaml.v3 v3.0.1 // indirect
```

Comment:
Please read our contribution guidelines, we require the algorithms to only use the standard library.